skrub/_table_vectorizer.py

```diff
@@ -812,6 +836,18 @@ def get_feature_names_out(self, input_features=None) -> list[str]:
 
         return all_trans_feature_names
 
+    @property
+    def named_transformers_(self):
+        return self._column_transformer.named_transformers_
+
+    @property
+    def sparse_output_(self):
+        return self._column_transformer.sparse_output_
+
+    @property
+    def output_indices_(self):
+        return self._column_transformer.output_indices_
+
```

Comment:
We need tests to check those attributes.

Reply:
What do we need these attributes for specifically?